mage/docker/lib.go

```diff
@@ -165,7 +166,12 @@ func Build(contextDir string, targetStage string, dockerFilePath string, namesAn
 		args = append(args, "--build-arg")
 		args = append(args, x)
 	}
-	return FromArgs(args)
+	c := FromArgs(args)
+	if c.Env == nil {
+		c.Env = map[string]string{}
+	}
+	c.Env["DOCKER_BUILDKIT"] = "1"
```

Comment:
The Dockerfile has been updated to use buildx which uses buildkit under the hood. In order to build locally using `docker build` we need to enable buildkit by setting this environment variable. Otherwise, `docker build` will have issues parsing the Dockerfile.